typescript/infra/config/environments/mainnet3/chains.ts

```diff
@@ -59,6 +59,12 @@ export type MainnetChains = keyof typeof mainnetConfigs;
 export const supportedChainNames = Object.keys(
   mainnetConfigs,
 ) as MainnetChains[];
+
+export const supportAgentChainNames = Object.keys({
```

Comment:
should this be used down below in `agentChainNames`? I don't see this used at all

Reply:
Yeah so it kinda isn't used, because I think agentChainNames IMO is only for hyperlane and rc contexts, and the neutron context manualy specifies the chains? I think this is another example where we have to do the improving by removing thing, since its very hard to know what impact removing neutron from nonEthereumMainnetConfigs has

Reply:
Undid the change